Composer/packages/client/src/pages/design/DebugPanel/DebugPanel.tsx

```diff
@@ -164,7 +152,24 @@ export const DebugPanel = () => {
         <div css={leftBarStyle} data-testid="statusbar__left">
           {headerPivot}
         </div>
-        <div css={rightBarStyle} data-testid="statusbar__right"></div>
+        <div
```

Comment:
This should complain that the role is missing on the div. Maybe use  button?

Reply:
It will. But We will run into css troubles if using other interactive elements. I think bearing the warning is acceptable considering the real problem here is the conflicts between UI design language (a wide blank bar) and UX behavior (click to expand).

Reply:
Could you elaborate? I think, u can use a role button. Disable the outline, adding a tabIndex.

Reply:
[eslint-plugin-jsx-a11y/no-static-element-interactions.md at master · jsx-eslint/eslint-plugin-jsx-a11y (github.com)](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-static-element-interactions.md#case-this-element-acts-like-a-button-link-menuitem-etc)